README.md

```diff
@@ -6,7 +6,7 @@ The Pony toolchain multiplexer
 
 [![Actions Status](https://github.com/ponylang/ponyup/workflows/vs-ponyc-latest/badge.svg)](https://github.com/ponylang/ponyup/actions)
 
-This project is currently in a pre-release beta state.
+This project is currently in beta software.
```

Comment:
I don't think this is grammatically correct. It should either be "This project is currently in beta" or "This project is currently beta software". I prefer the former.

Reply:
Fixed. merging.